components/hab/src/main.rs

```diff
@@ -942,7 +946,10 @@ fn install_sources_from_matches(matches: &ArgMatches) -> Result<Vec<InstallSourc
 }
 
 fn enable_features_from_env(ui: &mut UI) {
-    let features = vec![(feat::List, "LIST")];
+    let features = vec![
+        (feat::List, "LIST"),
+        (feat::OfflineInstall, "OFFLINE_INSTALL"),
+    ];
```

Comment:
Is there an alternative way to represent the features we're using? We've got the bitfield declared in one place, and the "meaning" of the flags defined here. Might be nice to have, say, a `features.rs` module where everything is consolidated in one easy-to-understand place.